jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.kt

```diff
@@ -90,34 +90,34 @@ class SendMessagePane(
     }
 
     private fun validateFields(): Boolean {
-        val inputIsValid = inputText.text.isNotEmpty()
-        if (!inputIsValid) {
-            runInEdt(ModalityState.any()) {
-                ComponentValidator.createPopupBuilder(ValidationInfo(message("sqs.message.validation.empty.message.body"), inputText), null)
-                    .setCancelOnClickOutside(true)
-                    .createPopup()
-                    .showUnderneathOf(inputText)
+        listOfNotNull(
+            if (inputText.text.isEmpty()) {
+                ValidationInfo(message("sqs.message.validation.empty.message.body"), inputText)
+            } else {
+                null
             }
-            return false
-        }
-
-        return if (queue.isFifo) {
-            val validationInfo = fifoFields.validateFields()
-            if (validationInfo == null) {
+        ).plus(
```

Comment:
if you are going to flatten lists  and use null filtering,

just do

`mutableListOf().apply {}`

Id be fine making this a util since we can't use the official buildList {} till 1.4 :(

Reply:
Sure, used `mutableListOf().apply`. We can make it a util if we ever use it again